src/pages/partners/deriv-prime/why-deriv-prime/why-deriv-prime.module.scss

```diff
@@ -0,0 +1,29 @@
+@use 'features/styles/theme/theme-mixins' as *;
```

Comment:
Similarly this file as features->pages->partners->deriv-prime->why-deriv-prime->why-deriv-prime.module.scss